server_admin/topics/authentication/flows.adoc

```diff
@@ -2,197 +2,228 @@
 
 === Authentication Flows
 
-An _authentication flow_ is a container for all authentications, screens, and actions that must happen during login, registration, and other
-{project_name} workflows.
-If you go to the admin console `Authentication` left menu item and go to the `Flows` tab, you can view all the defined flows
-in the system and what actions and checks each flow requires.
+An _authentication flow_ is a container of authentications, screens, and actions during login, registration, and other {project_name} workflows.
+To view all the flows, actions and checks each flow requires:
+
+.Procedure
+. Click *Authentication* in the menu.
+. Click the *Flows* tab.
 
 ==== Built-in flows
 
-{project_name} comes with a certain number of built-in flows. These flows cannot be modified, but the requirements can be modified to
-suit your needs.
+{project_name} has several built-in flows. It is not possible to modify these flows, but it is possible to alter the flow's requirements to suit your needs.
 
-This section does a walk-through of the built-in browser login flow.  In the
-left drop-down list select `browser` to come to the screen shown below:
+In the drop-down list, select *browser* to display the Browser Flow screen.
 
 .Browser Flow
-image:{project_images}/browser-flow.png[]
+image:{project_images}/browser-flow.png[Browser Flow]
+
+Hover over the question-mark tooltip of the drop-down list to view a description of the flow. Two sections exist.
+
+===== Auth type
+The name of the authentication or the action to execute. If an authentication is indented, it is in a sub-flow. It may or may not be executed, depending on the behavior of its parent. 
+
+. Cookie
++
+The first time a user logs in successfully, {project_name} sets a session cookie.  If the cookie is already set, this authentication type is successful. Since the cookie provider returned success and each execution at this level of the flow is _alternative_, {project_name} does not perform any other execution. This results in a successful login.
+
+. Kerberos
++
+This authenticator is disabled by default and is skipped during the Browser Flow.
+
+. Identity Provider Redirector
++
+This action is configured through the *Actions* > *Config*. It redirects to another IdP for <<_identity_broker, identity brokering>>.
```

Comment:
add "link" before the end of the first sentence.

Reply:
Not sure what this comment means. Are you saying all links should be link:{link}[{name}]? There are many example s of <<link, name>> which would have to change.